docs/import/add_disaster.js

```diff
@@ -70,8 +68,7 @@ function enableWhenReady() {
     });
 
     disasterPicker.on('change', () => toggleDisaster(disasterPicker.val()));
-    const mostRecent = querySnapshot.docs[querySnapshot.size - 1].id;
-    disasterPicker.val(mostRecent).trigger('change');
+    disasterPicker.val(getDisaster()).trigger('change');
```

Comment:
Something to consider here is that we are only selecting the current disaster after we've loaded all of the Firestore data. So we could modify getDisaster() to take a list of known disasters if available. That way if no current disaster is selected in localStorage, it could return the latest disaster, rather than the default of 2017-harvey. Not necessary for this PR, but something to maybe keep in mind.

Reply:
noted